vignettes/tidytranscriptomics.Rmd

```diff
@@ -150,18 +136,11 @@ We can use `select` to choose columns, for example, to see the sample, cell line
 airway %>% select(sample, cell, dex)
 ```
 
-We can use `group_by` and `summarise` to summarise information, for example to get the total counts for each sample.
-
-```{r}
-airway %>%
-    group_by(sample, cell, dex) %>%
-    summarise(total_counts=sum(counts))
-```
 We can use `mutate` to create a column. For example, we could create a new sample_name column that contains shorter sample names. We can remove the SRR1039 prefix that's present in all of the samples, as shorter names can fit better in some of the plots we will create. We can use `mutate` together with `str_replace` to remove the SRR1039 string from the sample column.
 
 ```{r}
 airway %>%
-  mutate(sample_name=str_remove(sample, "SRR1039")) %>%
+  mutate(sample_name=stringr::str_remove(sample, "SRR1039")) %>%
```

Comment:
Hello @mblue9 , in general I would think that loading the library would be cleaner than calling extemporaneously, also because user would know the list of libraries needed for the execution.

Any specific reason for this? Maybe I am missing some convenience.

Reply:
I've put it back. I read somewhere good to use :: for functions you only use once or twice and as we have the list of packages on the readme thought might change it but yes maybe better to leave it as library.

Reply:
> 
> 
> I've put it back. I read somewhere good to use :: for functions you only use once or twice and as we have the list of packages on the readme thought might change it but yes maybe better to leave it as library.

I see. Yes in this case it's true. The only thing that makes the code a little more busy. But now I don't have a strong opinion on in.

Reply:
I don't have a strong opinion on it either but think I'll stick with library for now.